content/doc/book/blueocean/pipeline-editor.adoc

```diff
@@ -18,130 +18,115 @@ endif::[]
+Selecting the icon:plus[] icon below an existing stage will add a parallel stage.
+* To delete unwanted stages, use the <<stage-configuration, more menu in the stage configuration pane>>.
 
-image:blueocean/editor/stage-editor-error.png[Stage editor with error, role=center]
+After setting the stage name and saving, the name displays above the stage.
+Stages that contain incomplete or invalid information display a icon:warning[].
+Pipelines can have validation errors during editing, but saving is blocked until the errors are fixed.
 
+image:blueocean/editor/stage-editor-error.png[Stage editor with error displayed., role=center]
 
 == Stage configuration
 
-Selecting a stage in the Stage editor will open the "Stage Configuration" sheet on the right side.
-Here we can change the name of the Stage, delete the Stage,
-and add steps to the Stage.
+Selecting a stage in the editor displays the *Stage Configuration* pane on the right side of the screen.
+Here, you can modify the name of the stage, delete the stage, and add steps to a stage.
 
-image:blueocean/editor/stage-configuration.png[Stage Configuration, role=center]
+image:blueocean/editor/stage-configuration.png[Stage configuration pane, role=center]
 
-The name of the Stage can be set at the top of the Stage Configuration sheet.
-The context menu (three dots on the upper right), can be used to delete the current stage.
-Clicking "Add step" will display the list of available Steps types with a search bar at the top.
-Steps can be deleted using the context <<step-configuration, context menu in the step configuration sheet>>.
-Adding a step or selecting an existing step will open the <<step-configuration, step configuration sheet>>.
+The name of the stage can be set at the top of the stage configuration pane.
+The more menu, represented by three dots to the right of the stage name, allows users to delete the currently selected stage.
+Selecting *Add step* displays the list of available step types.
+After selecting a step type, the page displays the step configuration pane.
 
 image:blueocean/editor/step-list.png[Step list filtered by 'file', role=center]
 
 == Step configuration
 
-Selecting a step from the Stage configuration sheet will open the Step Configuration sheet.
+This pane displays differently depending on the step type selected, and contains the fields or controls necessary for it.
 
 image:blueocean/editor/step-configuration.png[Step configuration for JUnit step, role=center]
 
-This sheet will differ depending on the step type,
-containing whatever fields or controls are needed.
-The name of the Step cannot be changed.
-The context menu (three dots on the upper right), can be used to delete the current step.
-Fields that contain incomplete or invalid information will display a warning symbol.
-Pipelines can have validation errors while they are being edited,
-but cannot be saved until the errors are fixed.
+After saving the name of the step, it will remain as originally entered.
```

Comment:
```suggestion
After saving the name of the step, it remains as originally configured.
```

Reply:
I wasn't fully happy with how I worded that sentence, so I rewrote that information to be a bit more clear. Would this work in its place, or would you have another suggestion on how to best navigate this info @cari-mason ? Thanks so much as always! 

```
Be sure to provide a strong name for the step, as the name retains its original configuration.
Deleting the step and recreating it, is the only way to provide a different name.
```